### DISCUSSION TITLE
Fprime on STM32H753-EVAL2 using FreeRTOS

### DISCUSSION THREAD

**Discussion by jck1044** [nasa/fprime#1746] [Q&A]
Hello all,

I am currently trying to get fprime on an STM32H753I-EVAL2 with FreeRTOS. I saw the thread https://github.com/nasa/fprime/discussions/978 where there seemed to be some success with the Nucleo-H743ZI2. I tried using that repo and changing it to work with my board, with no success. I am now in the beginning phases of the process and have some questions about getting started.

To my knowledge, fprime can be used without an OS, but we prefer to be able to use FreeRTOS. Using the STM32CubeIDE, I can configure the ioc file to set up FreeRTOS. Are there any special configurations that are necessary? Once it is installed, how do you know that it is working properly?

I was able to successfully install and run fprime on my Mac. Once FreeRTOS is configured properly on the STM32, how is fprime then installed on top of it?

Thanks,
Jared

**Comment by ThibFrgsGmz** [comment]
Were you able to get it to work? Were you able to get F´ running with `FreeRTOS` on your board?

**Reply by jck1044** [reply]
Not yet. I believe that I have FreeRTOS set up properly, and I am still trying to get F' running on top of it.

**Comment by BillYJT** [comment]
Hi! I started a more specific post regarding setup of fprime in FreeRTOS #1795 I thought it could be relevant so would like to mention it here.

**Reply by jck1044** [reply]
Thank you!!!
